generators/base/index.js

```diff
@@ -19,11 +19,24 @@ module.exports = class extends Generator {
         message: "Do you want i18n support?",
         default: true,
       },
-    ]).then(answers => {
+    ]).then((answers) => {
       this.i18nSupport = answers.i18nSupport;
     });
   }
 
+  prompting() {
+    return this.prompt([
+      {
+        type: "confirm",
+        name: "fastlane",
+        message: "Do you want add Fastlane to this project?",
+        default: true,
+      },
+    ]).then((answers) => {
+      this.fastlane = answers.fastlane;
+    });
+  }
+
```

Comment:
This doesn't feel quite right, should it be in the existing `prompting` function?

Reply:
Shout when it’s updated.